lambda/generate_metric_alarms/generate_metric_alarms.py

```diff
@@ -0,0 +1,132 @@
+"""Handler."""
+import base64
+import json
+import logging
+import os
+from collections import defaultdict
+
+from addict import Dict
+import boto3
+from botocore.exceptions import ClientError
+
+from exceptions import BadRequestError, ServerError
+import enrich
+
+LOG = logging.getLogger('generate_metric_alarms')
+
+
+def process_alert(event):
+    """Handles a new event request"""
+    return create_response(200, body='All good')
+
+
+def create_response(status_code, status_description='', body=''):
+    """Configure a response JSON object."""
+    response = {
+        "isBase64Encoded": False,
+        "headers": {
+            "Content-Type": "text/html;"
+        }
+    }
+
+    if not status_description:
+        description = {
+            200: '200 OK',
+            400: '400 Bad Request',
+            401: '401 Unauthorized',
+            405: '405 Method Not Allowed',
+            500: '500 Internal Server Error'
+        }
+        status_description = description.get(status_code)
+
+    response['statusCode'] = status_code
+    response['statusDescription'] = status_description
+    response['body'] = body
+
+    return response
+
+
+def create_sns_message(alert):
+    """Create the message to publish to SNS"""
+    try:
+        payload = json.dumps({"alert": alert})
+
+        message = json.dumps({
+            "default": "Default payload",
+            "sqs": payload,
+            "lambda": payload
+        })
+        LOG.debug('MESSAGE: %s', message)
+
+        return message
+    except TypeError as err:
+        raise ServerError('Error creating SNS message') from err
+
+
+def publish_alert(message, topic_arn):
+    """Publish alert message to SNS"""
+    try:
+        sns = boto3.client('sns')
+        sns_response = sns.publish(
+            TopicArn=topic_arn,
+            Message=message,
+            Subject='New report-phishing alert recorded',
```

Comment:
ctrl+c & ctrl+v :trollface:

Reply:
Yeah - I wanted something in place to make sure Ana's test framework was all working. I'm not even sure if this repo needs a publish to SNS method.